e2e/cypress/support/keycloak_commands.js

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import * as TIMEOUTS from '../fixtures/timeouts';
+
+const {
+    keycloakBaseUrl,
+    keycloakAppName,
+} = Cypress.env();
+
+const baseUrl = `${keycloakBaseUrl}/auth/admin/realms/${keycloakAppName}`;
+const loginUrl = `${keycloakBaseUrl}/auth/realms/master/protocol/openid-connect/token`;
+
+function buildProfile(user) {
+    return {
+        firstName: user.firstname,
+        lastName: user.lastname,
+        email: user.email,
+        username: user.username,
+        enabled: true,
+    };
+}
+
+Cypress.Commands.add('keycloakGetAccessTokenAPI', () => {
+    cy.task('keycloakRequest', {
```

Comment:
Please explicitly add `return`, especially for commands that's expected to return/wrap values. I had recent clean up related to it (see https://github.com/mattermost/mattermost-webapp/pull/7159/files)
```suggestion
    return cy.task('keycloakRequest', {
```